stubs/Modules/Base/App/Contacts/IsSearchable.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace App\Contacts;
+
+interface IsSearchable
+{
+    public function getName(): string;
```

Comment:
kun je misschien in een comment beschrijven wat deze methods doen of verwachten?